src/index.ts

```diff
@@ -25,4 +25,12 @@ export default class DataSource {
 
     return CD.get(url, options, cb);
   }
+
+  getAllRows(params: object) {
+    params['mi_multiple'] = true;
+
+    return this.getRawData(params).then(function(response) {
+      return JSON.parse(response.data);
```

Comment:
Looks fine to me, I haven't looked at the sorcerer change yet, but I thought we were just going to return an array as the top level payload?